spec/teacher_mailer_spec.rb

```diff
@@ -1,15 +1,15 @@
 require 'rails_helper'
 
 describe TeacherMailer do
-	fixtures :teachers
-	it 'Sends Welcome Email' do
+	fixtures :all
```

Comment:
https://stackoverflow.com/questions/6004057/with-rspec-how-to-seed-the-database-on-load/19930700#19930700